api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -208,15 +208,12 @@ public function doFromObject($dbObject, KalturaDetachedResponseProfile $response
 			$this->recordingOptions->fromObject($dbObject->getRecordingOptions());
 		}
 
-		if ($dbObject->getExplicitLive())
+		if ($dbObject->getViewMode() == ViewMode::ALLOW_ALL && in_array($dbObject->getLiveStatus(), array(KalturaEntryServerNodeStatus::PLAYABLE, KalturaEntryServerNodeStatus::BROADCASTING)))
 		{
-			if ($dbObject->getViewMode() == ViewMode::ALLOW_ALL && in_array($dbObject->getLiveStatus(), array(KalturaEntryServerNodeStatus::PLAYABLE, KalturaEntryServerNodeStatus::BROADCASTING)))
```

Comment:
Now that the object is no longer saved to the DB the cache won't expire on saving so, in my opinion, we need to reduce cache expiry (note this is an entry object and it is fetched quite a lot).